src/password/scripts/directives.js

```diff
@@ -3,9 +3,33 @@
 
     var module = angular.module('user_management.password');
 
+    module.service('catchErrors', [
```

Comment:
Hm, true. @maxpeterson wants this library to be more DRY but doing this would sacrifice the modularity of this lib.